From c32bc4a557fc66df0dbfead1bc4ecb7d1e4d342d Mon Sep 17 00:00:00 2001
From: Khiem Nguyen <khiem.nguyen.xt@renesas.com>
Date: Wed, 10 Mar 2021 18:24:13 +0700
Subject: [PATCH 26/40] drm: rcar-rvgc: drop _mode_ from
 drm_mode_connector_attach_encoder

The function name has been changed since v4.18 by commit
cde4c44d (drm: drop _mode_ from drm_mode_connector_attach_encoder).
This patch changes the function name accordingly.

Signed-off-by: Khiem Nguyen <khiem.nguyen.xt@renesas.com>
---
 drivers/gpu/drm/rcar-rvgc/rcar_rvgc_pipe.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/rcar-rvgc/rcar_rvgc_pipe.c b/drivers/gpu/drm/rcar-rvgc/rcar_rvgc_pipe.c
index dd1dadf5a9ee..470b599abf9e 100644
--- a/drivers/gpu/drm/rcar-rvgc/rcar_rvgc_pipe.c
+++ b/drivers/gpu/drm/rcar-rvgc/rcar_rvgc_pipe.c
@@ -481,7 +481,7 @@ static int overlay_pipe_init(struct rcar_rvgc_device* rvgc_dev,
 		return ret;
 	connector = rvgc_connector_create(rvgc_pipe);
 
-	return drm_mode_connector_attach_encoder(connector, encoder);
+	return drm_connector_attach_encoder(connector, encoder);
 
 }
 
-- 
2.25.1

